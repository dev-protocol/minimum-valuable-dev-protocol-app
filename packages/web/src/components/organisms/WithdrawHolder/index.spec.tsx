import React from 'react'
import { render } from '@testing-library/react'
import { WithdrawHolderCard } from './'

jest.mock('src/fixtures/dev-kit/hooks')

describe(`${WithdrawHolderCard.name}`, () => {
  test('Snapshot', () => {
    const component = render(<WithdrawHolderCard propertyAddress="propertyAddress" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
