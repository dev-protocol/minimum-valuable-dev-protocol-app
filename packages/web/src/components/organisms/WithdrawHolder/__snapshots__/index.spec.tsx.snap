// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithdrawHolderCard Snapshot 1`] = `
<body>
  <div>
    <div
      class="ant-card ant-card-bordered"
    >
      <div
        class="ant-card-head"
      >
        <div
          class="ant-card-head-wrapper"
        >
          <div
            class="ant-card-head-title"
          >
            Withdraw Holder Reward
          </div>
          <div
            class="ant-card-extra"
          >
            <p>
              Last Update: 
              42141412412
            </p>
            <a
              href="#"
              rel="noreferrer noopener"
              target="_blank"
            >
              Mining now
            </a>
          </div>
        </div>
      </div>
      <div
        class="ant-card-body"
      >
        <p>
          6000
           DEV
        </p>
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Withdraw
          </span>
        </button>
      </div>
    </div>
  </div>
</body>
`;
